packages/standard-minifier-js/plugin/minify-js.js

```diff
@@ -9,97 +38,202 @@ Plugin.registerMinifier({
 
 class MeteorMinifier {
 
-  async processFilesForBundle (files, options) {
-    const mode = options.minifyMode;
-
-    // don't minify anything for development
-    if (mode === 'development') {
-      files.forEach(function (file) {
-        file.addJavaScript({
-          data: file.getContentsAsBuffer(),
-          sourceMap: file.getSourceMap(),
-          path: file.getPathInBundle(),
-        });
-      });
-      return;
-    }
+  constructor() {
+    this.config = getConfig();
+  }
 
-    // this function tries its best to locate the original source file
-    // that the error being reported was located inside of
-    function maybeThrowMinifyErrorBySourceFile(error, file) {
+  _minifyWithSWC(file) {
+    swc = swc || require('@meteorjs/swc-core'); 
+    const NODE_ENV = process.env.NODE_ENV || 'development';
+    
+    let map = file.getSourceMap();
```

Comment:
If it's not using the output source map, it would be faster to not provide the source map to swc.